src/jsonToCsv.ts

```diff
@@ -0,0 +1,67 @@
+import { createReadStream, createWriteStream } from "fs";
+import { stringifyJsonChunk, prepareRow } from "./utils";
+import { Header, Paths, RowObject } from "./types";
+import JSONStream from "jsonstream";
```

Comment:
Could please don't use this library

Reply:
The second function uses regexs and is consistent with our assumptions that I should not use any library.
Only one function uses these libraries and it is a result of my playing around and testing stuff. I simply decided to leave it